### PR TITLE
Correctly detect repositories

### PR DESCRIPTION
**Pull request by adrianreber**

### CODE REVIEW

File: utility/mm2_update-master-directory-list [hunk 1]
```diff
@@ -290,19 +290,6 @@ def make_repo_file_details(session, diskpath, relativeDName, D, category, target
         session.commit()
 
 
-def move_repository_from_development(prefix, arch, newdir):
```
Comment (pypingou) [inline utility/mm2_update-master-directory-list]: was this not called anywhere?

**Comment by pypingou** [comment]
One question otherwise looks fine to me :)

File: utility/mm2_update-master-directory-list (hunk 1 above)
Comment (adrianreber) [reply utility/mm2_update-master-directory-list]: This function has never been used. Even going back to the version 0.0.1 tag it is not used anywhere.

This step is always done manually after a release.

Comment (pypingou) [reply utility/mm2_update-master-directory-list]: Alright :)

**Pull request merged**